ganga/GangaCore/Utility/Config/__init__.py

```diff
@@ -1,5 +1,5 @@
 
-from .Config import getConfig, makeConfig, ConfigError, setSessionValuesFromFiles, allConfigs, setConfigOption, expandConfigPath, config_scope, setSessionValue, setUserValue
+from .Config import getConfig, makeConfig, ConfigError, setSessionValuesFromFiles, allConfigs, setConfigOption, expandConfigPath, config_scope, setSessionValue, setUserValue, setUserValueForTest
```

Comment:
Many of these functions are not used in the code below. Why import them here?

Reply:
I have just appended `setUserValueForTest` to the list, for `from GangaCore.Utility.Config import setUserValueForTest`